source/plugins/go/src/oms.go

```diff
@@ -1116,6 +1122,62 @@ func UpdateNumTelegrafMetricsSentTelemetry(numMetricsSent int, numSendErrors int
 	ContainerLogTelemetryMutex.Unlock()
 }
 
+func processIncludes(kubernetesMetadataMap map[string]interface{}, includesList []string) map[string]interface{} {
+	includedMetadata := make(map[string]interface{})
+	for _, include := range includesList {
+		switch include {
+		case "podUid":
```

Comment:
we should convert the include fields in lower case and use the lower case for comparison so that  we can support even if customer having different casing for the field names in include_list

Reply:
The name in this format is to align with KubePodInventory table,
Though we can change to case from case "podUid"  to  case "poduid"